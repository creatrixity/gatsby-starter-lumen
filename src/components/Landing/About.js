// @flow strict
import React, { useRef } from 'react';
import cx from 'classnames';
import { useIntersectionObserver } from '../../hooks';
import styles from './Landing.module.scss';

type Props = {
  children: React.ReactChildren,
};

const About = ({ children }: Props) => {
  const aboutRef = useRef();
  const entry = useIntersectionObserver(aboutRef, { freezeOnceVisible: true });
  const animateAboutTextIn = !!entry?.isIntersecting;

  return (
    <section className={cx(styles.About)} ref={aboutRef} id="about">
      <section className={cx([styles.container, styles['container--fullHeight']])}>
        <div className={styles.landing__About}>
          <section>
            <h1
              className={cx([styles.About__title, { 'animate__fadeInUp animate__animated': animateAboutTextIn }])}
              data-label="01."
            >
              About me
            </h1>
            <p
              className={cx([
                styles.About__intro,
                { 'animate__fadeInUp animate__animated animate__delay-1s': animateAboutTextIn },
              ])}
            >
              Iâ€™m Caleb (sometimes referred to as Kay) and I enjoy solving problems through code and design. I took an
              interest in web development in the aftermath of interest in gaming and the first moment I made Sonic run
              on my little 65k color Nokia mobile browser way back in early 2008 was magical for me.
            </p>
            <p
              className={cx([
                styles.About__intro,
                { 'animate__fadeInUp animate__animated animate__delay-1s': animateAboutTextIn },
              ])}
            >
              Over the last couple of years, I've had the good fortune of working on a variety of interesting projects,
              from building terminal management systems for POS devices to onboarding software for the military. My main
              focus these days is building products that help us solve more problems for financially underrepresented
              people at Kudi.{' '}
            </p>
            <p
              className={cx([
                styles.About__intro,
                { 'animate__fadeInUp animate__animated animate__delay-2s': animateAboutTextIn },
              ])}
            >
              Recently, I've taken a keener interest in sharing my knowledge and you really should{' '}
              <a className="with-underline" href="/articles">
                read my blog
              </a>{' '}
              where I write about React and Node.js.
            </p>
          </section>
        </div>
      </section>
    </section>
  );
};

export default About;
