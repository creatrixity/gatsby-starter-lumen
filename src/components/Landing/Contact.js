// @flow strict
import React, { useRef } from 'react';
import cx from 'classnames';
import { useIntersectionObserver } from '../../hooks';
import styles from './Landing.module.scss';

type Props = {
  children: React.ReactChildren,
};

const Contact = ({ children }: Props) => {
  const contactRef = useRef();
  const entry = useIntersectionObserver(contactRef, { freezeOnceVisible: true });
  const animateContactTextIn = !!entry?.isIntersecting;

  return (
    <section className={cx(styles.Contact)} ref={contactRef}>
      <section className={cx([styles.container, styles['container--fullHeight']])}>
        <div className={styles.landing__Contact}>
          <section>
            <h1
              className={cx([styles.Contact__title, { 'animate__fadeInUp animate__animated': animateContactTextIn }])}
              data-label="04."
            >
              Let's chat.
            </h1>
            <p
              className={cx([
                styles.Contact__intro,
                { 'animate__fadeInUp animate__animated animate__delay-1s': animateContactTextIn },
              ])}
            >
              Got a question or proposal for me? Letâ€™s get in touch! I have a personal rule of returning responses
              within a 3-hour window.
            </p>
            <p
              className={cx([
                styles.Contact__intro,
                { 'animate__fadeInUp animate__animated animate__delay-2s': animateContactTextIn },
              ])}
            >
              Start by{' '}
              <a className="with-underline" href="#">
                saying hi
              </a>{' '}
            </p>
          </section>
        </div>
      </section>
    </section>
  );
};

export default Contact;
