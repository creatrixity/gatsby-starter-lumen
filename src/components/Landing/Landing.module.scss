@import '../../assets/scss/variables';
@import '../../assets/scss/mixins';

.landing {
  height: 100%;
  width: 100%;
  display: block;

  &__overlay {
    content: '';
    position: fixed;
    display: none;
    top: 0;
    left: 0;
    height: 100vh;
    width: 100vw;
    background: rgba(0, 0, 0, 0.64);

    &__open {
      display: block;
    }
  }

  &__header {
    display: flex;
    align-items: center;
    justify-content: space-between;
    padding: 4rem;
    color: $color-primary;

    @include breakpoint-xs {
      width: 100%;
      padding: 3rem 0;
    }
  }

  &__logo {
    width: 150;
  }

  &__nav__container {
    @include respond-to('small') {
      display: none;
    }
  }

  &__About,
  &__Contact {
    display: flex;
    align-items: center;
    min-height: 100vh;
  }
}

.container {
  lost-center: $layout-width;

  &--fullHeight {
    min-height: 100vh;
  }
}

.MobileMenu {
  display: none;
  transition: transform 0.3s cubic-bezier(0.55, 0.055, 0.675, 0.19);

  @include respond-to('small') {
    display: flex;
    -webkit-box-pack: center;
    justify-content: center;
    -webkit-box-align: center;
    align-items: center;
    position: fixed;
    top: 0px;
    bottom: 0px;
    right: 0px;
    padding: 50px 10px;
    width: 75vw;
    height: 100vh;
    outline: 0px;
    background-color: $color-tertiary;
    box-shadow: -10px 0px 30px -15px $color-secondary;
    z-index: 9;
    transform: translateX(100vw);
    visibility: hidden;

    nav {
      display: flex;
      -webkit-box-pack: justify;
      justify-content: space-between;
      -webkit-box-align: center;
      align-items: center;
      width: 100%;
      flex-direction: column;
      color: var(--lightest-slate);
      font-family: var(--font-mono);
      text-align: center;
    }

    ol {
      padding: 0px;
      margin: 0 0 1rem 0;
      list-style: none;
      width: 100%;
    }

    li {
      margin: 0px auto 10px;

      &::before {
        counter-increment: item;
        content: '0' counter(item) '.';
        display: block;
        margin-bottom: 5px;
        color: $color-primary;
        font-size: var(--fz-sm);
      }

      a {
        display: inline-block;
        text-decoration: none;
        text-decoration-skip-ink: auto;
        position: relative;
        font-size: $type-scale-6;
        color: rgba(255, 255, 255, 0.64);
        transition: var(--transition);
        width: 100%;
        padding: 3px 20px 20px;

        &:before {
        }
      }
    }
  }

  &__open {
    transform: translateX(0vw);
    visibility: visible;
  }
}

.Hamburger {
  &,
  &__button {
    display: none;
  }

  @include respond-to('small') {
    display: inline-block;
    position: relative;
    width: $hamburger-width;
    height: 24px;

    &__button {
      display: flex !important;
      -webkit-box-pack: center;
      justify-content: center;
      -webkit-box-align: center;
      align-items: center;
      position: relative;
      z-index: 10;
      margin-right: -15px;
      padding: 15px;
      border: 0px;
      background-color: transparent;
      color: inherit;
      text-transform: none;
      transition-timing-function: linear;
      transition-duration: 0.15s;
      transition-property: opacity, filter;

      &:focus {
        outline: 2px dashed currentColor;
        outline-offset: 3px;
      }
    }

    &__closed &__inner {
      transform: rotate(225deg);

      &:before {
        opacity: 0;
      }

      &:after {
        transform: rotate(-90deg);
        bottom: 0;
      }
    }

    &__inner {
      position: absolute;
      top: 50%;
      right: 0px;
      width: $hamburger-width;
      height: 2px;
      border-radius: 2px;
      background-color: currentColor;
      transition: transform 0.22s cubic-bezier(0.55, 0.055, 0.675, 0.19) 0s;
      transform: rotate(0deg);

      &::before,
      &::after {
        content: '';
        display: block;
        position: absolute;
        left: auto;
        right: 0px;
        width: $hamburger-width;
        height: 2px;
        border-radius: 4px;
        background-color: currentColor;
      }

      &:before {
        width: 120%;
        top: -10px;
        opacity: 1;
        transition: var(--ham-before);
      }
    }
  }
}

.Hero {
  min-height: 100vh;
  background: $color-secondary;
  display: flex;
  flex-direction: column;
  align-items: center;

  @include breakpoint-xs {
    padding: 0 2rem 2rem;
  }

  &__headingGroup {
    margin-bottom: 3rem;

    @include respond-to('small') {
      margin-bottom: 1rem;
    }
  }

  &__title {
    color: #fff;
  }
  &__subtitle {
    color: rgba(255, 255, 255, 0.52);
  }
  &__intro,
  &__text {
    color: rgba(255, 255, 255, 0.36);
  }

  &__text {
    font-size: $type-scale-5;
    font-weight: 600;

    @include respond-to('small') {
      display: none;
    }
  }

  &__intro {
    color: rgba(255, 255, 255, 0.6);
    font-size: $type-scale-4;
    line-height: $type-scale-3;
    width: 70%;
    margin-bottom: 3rem;

    @include respond-to('small') {
      font-size: $type-scale-5;
      line-height: $type-scale-4;
      width: 100%;
    }
  }
}

.NavList {
  display: inline-flex;
  list-style: none;

  @include respond-to('small') {
    display: none;
  }

  &,
  &__item {
    margin: 0;
    padding: 0;
  }

  &__link {
    color: rgba(255, 255, 255, 0.36);
    font-size: $type-scale-5;
    display: flex;
    align-items: center;
    padding: 1rem 2rem;
    transition: color 1s cubic-bezier(0.165, 0.84, 0.44, 1);

    &:hover {
      color: $color-primary;

      & .NavList__link__icon {
        transform: translateX(-4px);
      }
    }
  }

  &__link__icon {
    display: block;
    margin: 0 0.5rem;
    transition: transform 1s cubic-bezier(0.23, 1, 0.32, 1);
  }
}

.About {
  background: $color-tertiary;
  @include respond-to('small') {
    & {
      padding: 5rem 2rem;
    }

    &__intro {
      width: 100% !important;
    }
  }

  &__title {
    color: #fff;
    text-align: right;
    margin-bottom: 4rem;

    &::before {
      content: attr(data-label);
      display: block;
      color: rgba(255, 255, 255, 0.16);
      padding: 0;
      line-height: $type-scale-3;
    }
  }

  &__intro {
    color: rgba(255, 255, 255, 0.64);
    font-size: $type-scale-5;
    line-height: $type-scale-4;
    width: 70%;
  }
}

.Contact {
  background: $color-secondary;

  &__title {
    color: #fff;
    margin-bottom: 4rem;
    text-align: right;

    &::before {
      content: attr(data-label);
      display: block;
      color: rgba(255, 255, 255, 0.16);
      padding: 0;
      line-height: $type-scale-3;
    }
  }

  &__intro {
    color: rgba(255, 255, 255, 0.36);
    font-size: $type-scale-3;
    line-height: $type-scale-2;
    width: 70%;
  }

  @include respond-to('small') {
    padding: 0 1rem;

    &__title {
      margin-bottom: 1rem;
      text-align: left;
    }

    &__intro {
      font-size: $type-scale-5;
      line-height: $type-scale-4;
    }
  }
}

.Skills,
.Stints {
  display: flex;

  &__title {
    color: #fff;
    margin-bottom: 4rem;

    &::before {
      content: attr(data-label);
      display: block;
      color: rgba(255, 255, 255, 0.16);
      padding: 0;
      line-height: $type-scale-3;
    }
  }

  &__list,
  &__details {
    flex: 1;
    display: flex;
    align-items: center;
    box-sizing: border-box;
    min-height: 100%;
    padding: 0;
    margin: 0;
    margin: 0;
  }

  &__list {
    background: darken($color-tertiary, 1%);
    box-sizing: border-box;

    &__inner {
      padding: 8rem;
      width: 100%;
    }
  }
  &__details {
    position: relative;
    background: $color-secondary;
    overflow-x: hidden;
  }

  &__wrapper {
    height: 100%;
    transition: transform 0.5s cubic-bezier(0.55, 0.055, 0.675, 0.19);
  }

  &__detail {
    position: fixed;
    width: 100%;
    top: 0;
    bottom: 0;
    padding: 8rem;
    z-index: 0;
    display: flex;
    align-items: center;

    &__title {
      opacity: 0;
    }

    &__subtitle {
      opacity: 0;
      font-weight: bold;
      text-align: right;
    }

    &__desc {
      opacity: 0;
      color: rgba(255, 255, 255, 0.64);
      line-height: $type-scale-3;
      font-size: $type-scale-4;
      color: rgba(0, 0, 0, 0.64);
    }

    &__javascript {
      background-color: #efdb4f;
    }

    &__nodejs {
      background-color: #84cd29;
    }
    &__react {
      background-color: #61dafb;
    }
    &__typescript {
      background-color: #007acc;
    }
    &__python {
      background-color: #ffdc51;
    }
    &__rust {
      background-color: #99f1ff;
    }
  }
}

.Skills,
.Stints {
  @include respond-to('small') {
    flex-direction: column;

    .Hamburger__button {
      color: $color-tertiary;
      top: 1rem;
      right: 3rem;
      position: absolute;
    }

    &__detail {
      position: absolute;
      padding: 2rem;
      font-size: $type-scale-6;

      &__desc {
        line-height: $type-scale-4;
        font-size: $type-scale-5;
      }
    }

    &__details {
      display: flex;
      -webkit-box-pack: center;
      justify-content: center;
      -webkit-box-align: center;
      align-items: center;
      position: fixed;
      top: 0px;
      bottom: 0px;
      right: 0px;
      width: 100vw;
      outline: 0px;
      background-color: $color-tertiary;
      box-shadow: -10px 0px 30px -15px $color-secondary;
      z-index: 100;
      transition: transform 0.3s ease;
      transform: translateX(100vw);
      visibility: hidden;

      &__open {
        transform: translateX(0);
        visibility: visible;
      }
    }

    &__list {
      display: block;

      &__inner {
        padding: 2rem;
      }
    }
  }
}

.Stints {
  &__icon {
    max-width: 150px;
  }

  @include respond-to('small') {
    &__icon {
      max-width: 75px;
    }
  }

  &__detail {
    &__title {
      color: $color-primary;
      opacity: 0;
    }

    &__desc {
      opacity: 0;
      color: rgba(255, 255, 255, 0.64);
      font-size: $type-scale-3;
      font-weight: 600;
    }

    &__epoch {
      font-size: $type-scale-4;
      color: rgba(255, 255, 255, 0.36);
      opacity: 0;
    }

    &__highlights {
      list-style: none;
      padding: 0;
    }

    &__highlight {
      color: rgb(255, 255, 255);
      padding: 10px;
      font-size: $type-scale-5;
      display: flex;
      align-items: top;
      opacity: 0;
    }

    &__highlight__icon {
      margin: 0.5rem 1rem 0 0;
    }
  }

  @include respond-to('small') {
    &__detail {
      background: $color-secondary;

      .Hamburger__button {
        color: $color-primary;
      }

      &__desc {
        font-size: $type-scale-5;
      }

      &__epoch {
        font-size: $type-scale-6;
      }

      &__highlight {
        font-size: $type-scale-6;
      }
    }
  }
}

.Contact {
  @include respond-to('small') {
    &__intro {
      font-size: $type-scale-6;
      width: 100%;
    }
  }

  &__footer {
    @include respond-to('small') {
      display: block;
      font-size: $type-scale-6;
      padding: 2rem 0;
    }

    display: flex;
    align-items: center;
    justify-content: space-between;
    padding: 4rem 0;
    font-size: $type-scale-5;
    color: rgba(255, 255, 255, 0.36);

    &__link {
      display: flex;
      align-items: center;
    }
  }
}
