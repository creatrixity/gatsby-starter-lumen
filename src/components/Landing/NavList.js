// @flow strict
import React from 'react';
import cx from 'classnames';
import { getIcon } from '../../utils';
import Icon from '../Icon/Icon';
import styles from './Landing.module.scss';

type NavItem = {
  label: string,
  path: string,
};

type Props = {
  navItems: NavItem[],
  animateLinksIn: Boolean,
};

const NavList = ({ navItems, animateLinksIn }: Props) =>
  navItems.length ? (
    <ul className={styles.NavList}>
      {navItems.map(({ label, path }: NavItem, idx) => {
        const animateClasses = `animate__fadeInDown animate__animated animate__delay-${idx + 1}s`;
        return (
          <li key={label} className={cx(styles.NavList__item, { [animateClasses]: animateLinksIn })}>
            <a className={styles.NavList__link} href={path}>
              <Icon
                name="caret"
                icon={getIcon('caretRight')}
                width="11"
                height="13"
                fill="none"
                className={styles.NavList__link__icon}
              />
              {label}
            </a>
          </li>
        );
      })}
    </ul>
  ) : null;

export default NavList;
