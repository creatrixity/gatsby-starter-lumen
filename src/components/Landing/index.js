// @flow strict
export { default } from './Landing';
