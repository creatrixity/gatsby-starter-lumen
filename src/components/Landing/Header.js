// @flow strict
import React from 'react';
import cx from 'classnames';
import { useSiteMetadata } from '../../hooks';
import { getIcon } from '../../utils';
import Icon from '../Icon/Icon';
import styles from './Landing.module.scss';
import NavList from './NavList';

type Props = {
  name: string,
  icon: {
    viewBox?: string,
    path?: string,
  },
};

const Header = () => {
  const { menu } = useSiteMetadata();

  return (
    <section className={styles.landing__header}>
      <Icon
        useDefaultStyles={false}
        name="caret"
        icon={getIcon('logo')}
        width="59"
        height="40"
        fill="none"
        className={cx([styles.landing__logo, 'animate__fadeInDown animate__animated animate__delay-1s'])}
      />

      <section>
        <NavList navItems={menu} animateLinksIn={true} />
        <a className="btn-primary animate__fadeInDown animate__animated animate__delay-5s">My Resume</a>
      </section>
    </section>
  );
};

export default Header;
