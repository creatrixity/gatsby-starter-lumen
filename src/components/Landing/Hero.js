// @flow strict
import React, { useRef } from 'react';
import cx from 'classnames';
import { useIntersectionObserver } from '../../hooks';
import styles from './Landing.module.scss';
import Header from './Header';

type Props = {
  children: React.ReactChildren,
};

const Hero = ({ children }: Props) => {
  const heroRef = useRef();
  const entry = useIntersectionObserver(heroRef, { freezeOnceVisible: true });
  const animateHeroTextIn = !!entry?.isIntersecting;

  return (
    <section className={styles.Hero}>
      <Header animateLinksIn={animateHeroTextIn} />

      <section className={cx([styles.container])} ref={heroRef}>
        <p
          className={cx([
            styles.Hero__text,
            { 'animate__fadeInUp animate__animated animate__delay-4s': animateHeroTextIn },
          ])}
        >
          Hello pardner,
        </p>
        <hgroup className={styles.Hero__headingGroup}>
          <h1
            className={cx([
              styles.Hero__title,
              { 'animate__fadeInUp animate__animated animate__delay-5s': animateHeroTextIn },
            ])}
          >
            I’m Caleb Mathew.
          </h1>
          <h2
            className={cx([
              styles.Hero__subtitle,
              { 'animate__fadeInUp animate__animated animate__delay-5s': animateHeroTextIn },
            ])}
          >
            I help improve mankind’s greatest invention.
          </h2>
        </hgroup>
        <p
          className={cx([
            styles.Hero__intro,
            { 'animate__fadeInUp animate__animated animate__delay-5s': animateHeroTextIn },
          ])}
        >
          I’m a software engineer that enjoys designing and building helpful products. My current adventure is building
          financial products at{' '}
          <a className="with-underline" href="https://kudi.ai">
            Kudi
          </a>
        </p>
        <a
          href={'/articles'}
          className="btn-primary btn-primary--xxl animate__fadeInUp animate__animated animate__delay-5s"
        >
          See my articles
        </a>
      </section>
    </section>
  );
};

export default Hero;
