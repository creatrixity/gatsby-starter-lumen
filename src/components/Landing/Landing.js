// @flow strict
import React from 'react';
import Hero from './Hero';
import Header from './Header';
import styles from './Landing.module.scss';

type Props = {
  name: string,
  icon: {
    viewBox?: string,
    path?: string,
  },
};

const Landing = () => (
  <section className={styles.landing}>
    <Hero>
      <Header />
    </Hero>
  </section>
);

export default Landing;
