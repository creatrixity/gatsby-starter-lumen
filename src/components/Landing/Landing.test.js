// @flow strict
import React from 'react';
import renderer from 'react-test-renderer';
import Landing from './Landing';

describe('Landing', () => {
  const props = {};

  it('renders correctly', () => {
    const tree = renderer.create(<Landing {...props} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
