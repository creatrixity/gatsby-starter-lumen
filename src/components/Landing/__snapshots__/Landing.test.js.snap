// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing renders correctly 1`] = `
<section
  className="landing"
>
  <section
    className="Hero"
  >
    <section
      className="landing__header"
    >
      <svg
        className="landing__logo animate__fadeIn animate__animated animate__delay-2s"
        fill="none"
        height="40"
        viewBox="0 0 76 40"
        width="59"
      >
        <title>
          caret
        </title>
        <path
          d="M74.2402 18.0147L58.0882 1.86275C56.7647 0.539217 54.6078 0.539217 53.2843 1.86275L23.3824 31.7647L12.0588 20.5637L32.9902 0H23.2843L6.37255 15.6373V0H0V39.7059H6.37255V25.2451L20.5392 38.9951C21.201 39.6569 22.2794 40 23.1618 40C24.0441 40 25 39.6569 25.6618 38.9951L55.6373 9.09314L67.0098 20.2941L47.4755 39.7059H57.0833L74.2402 22.6961C74.8775 22.0588 75.2451 21.2745 75.2451 20.3676C75.2206 19.4608 74.8775 18.652 74.2402 18.0147Z"
          fill="#99F0FF"
        />
        <path
          d="M58.9216 17.1569C57.5735 15.8333 55.4412 15.7108 54.1176 17.0343L31.5931 39.7059H41.152L58.9461 21.8382C60.2696 20.4902 60.2451 18.4804 58.9216 17.1569Z"
          fill="#99F0FF"
        />
      </svg>
      <section
        className="landing__nav__container"
      >
        <ul
          className="NavList"
        >
          <li
            className="NavList__item"
          >
            <a
              className="NavList__link"
              href="/test/1/"
            >
              <svg
                className="NavList__link__icon"
                fill="none"
                height="13"
                viewBox="0 0 11 13"
                width="11"
              >
                <title>
                  caret
                </title>
                <path
                  d="M8.95837 8.88236C10.3138 7.68793 10.3138 5.57523 8.95837 4.3808L6.48343 2.19984C4.54592 0.492473 1.5 1.86816 1.5 4.45062L1.5 8.81254C1.5 11.395 4.54592 12.7707 6.48343 11.0633L8.95837 8.88236Z"
                  fill="none"
                  stroke="#99F0FF"
                  strokeWidth={2}
                />
              </svg>
              Test label 1
            </a>
          </li>
          <li
            className="NavList__item"
          >
            <a
              className="NavList__link"
              href="/test/2/"
            >
              <svg
                className="NavList__link__icon"
                fill="none"
                height="13"
                viewBox="0 0 11 13"
                width="11"
              >
                <title>
                  caret
                </title>
                <path
                  d="M8.95837 8.88236C10.3138 7.68793 10.3138 5.57523 8.95837 4.3808L6.48343 2.19984C4.54592 0.492473 1.5 1.86816 1.5 4.45062L1.5 8.81254C1.5 11.395 4.54592 12.7707 6.48343 11.0633L8.95837 8.88236Z"
                  fill="none"
                  stroke="#99F0FF"
                  strokeWidth={2}
                />
              </svg>
              Test label 2
            </a>
          </li>
          <li
            className="NavList__item"
          >
            <a
              className="NavList__link"
              href="/test/3/"
            >
              <svg
                className="NavList__link__icon"
                fill="none"
                height="13"
                viewBox="0 0 11 13"
                width="11"
              >
                <title>
                  caret
                </title>
                <path
                  d="M8.95837 8.88236C10.3138 7.68793 10.3138 5.57523 8.95837 4.3808L6.48343 2.19984C4.54592 0.492473 1.5 1.86816 1.5 4.45062L1.5 8.81254C1.5 11.395 4.54592 12.7707 6.48343 11.0633L8.95837 8.88236Z"
                  fill="none"
                  stroke="#99F0FF"
                  strokeWidth={2}
                />
              </svg>
              Test label 3
            </a>
          </li>
        </ul>
        <a
          className="btn-primary animate__fadeInDown animate__animated animate__delay-5s"
          target="_blank"
        >
          My Resume
        </a>
      </section>
      <button
        aria-label="menu"
        className="Hamburger__button"
        onClick={[Function]}
      >
        <span
          className="Hamburger"
        >
          <span
            className="Hamburger__inner"
          />
        </span>
      </button>
      <aside
        aria-hidden="false"
        className="MobileMenu"
        tabIndex="1"
      >
        <nav>
          <ol>
            <li>
              <a
                href="/test/1/"
              >
                Test label 1
              </a>
            </li>
            <li>
              <a
                href="/test/2/"
              >
                Test label 2
              </a>
            </li>
            <li>
              <a
                href="/test/3/"
              >
                Test label 3
              </a>
            </li>
          </ol>
          <a
            className="btn-primary animate__fadeInDown animate__animated animate__delay-5s"
            target="_blank"
          >
            My Resume
          </a>
        </nav>
      </aside>
    </section>
    <section
      className="container"
    >
      <p
        className="Hero__text"
      >
        Hello pardner,
      </p>
      <hgroup
        className="Hero__headingGroup"
      >
        <h1
          className="Hero__title"
        >
          I’m Caleb Mathew.
        </h1>
        <h2
          className="Hero__subtitle"
        >
          I help improve mankind’s greatest invention.
        </h2>
      </hgroup>
      <p
        className="Hero__intro"
      >
        I’m a software engineer that enjoys designing and building helpful products. My current adventure is building financial products at
         
        <a
          className="with-underline"
          href="https://kudi.ai"
        >
          Kudi
        </a>
      </p>
      <a
        className="btn-primary btn-primary--xxl animate__fadeInUp animate__animated animate__delay-5s"
        href="/articles"
      >
        See my articles
      </a>
    </section>
  </section>
  <section
    className="About"
    id="about"
  >
    <section
      className="container container--fullHeight"
    >
      <div
        className="landing__About"
      >
        <section>
          <h1
            className="About__title"
            data-label="01."
          >
            About me
          </h1>
          <p
            className="About__intro"
          >
            I’m Caleb (sometimes referred to as Kay) and I enjoy solving problems through code and design. I took an interest in web development in the aftermath of interest in gaming and the first moment I made Sonic run on my little 65k color Nokia mobile browser way back in early 2008 was magical for me.
          </p>
          <p
            className="About__intro"
          >
            Over the last couple of years, I've had the good fortune of working on a variety of interesting projects, from building terminal management systems for POS devices to onboarding software for the military. My main focus these days is building products that help us solve more problems for financially underrepresented people at Kudi.
             
          </p>
          <p
            className="About__intro"
          >
            Recently, I've taken a keener interest in sharing my knowledge and you really should
             
            <a
              className="with-underline"
              href="/articles"
            >
              read my blog
            </a>
             
            where I write about React and Node.js.
          </p>
        </section>
      </div>
    </section>
  </section>
  <section
    className="Skills"
    id="skills"
  >
    <div
      className="Skills__list"
    >
      <section
        className="Skills__list__inner"
      >
        <h1
          className="Skills__title"
          data-label="02."
        >
          Skills.
        </h1>
        <section
          className="Entries"
        >
          <div>
            <input
              checked={true}
              className="EntryCard__radio"
              id="entry-javascript"
              onChange={[Function]}
              type="radio"
            />
            <label
              className="EntryCard"
              htmlFor="entry-javascript"
            >
              <section>
                <h4
                  className="EntryCard__title"
                >
                  JavaScript
                </h4>
                <a
                  className="EntryCard__link with-underline"
                  href="#"
                >
                  Read more
                </a>
              </section>
              <img
                src="https://cdn.jsdelivr.net/gh/devicons/devicon/icons/javascript/javascript-original.svg"
                width="64"
              />
            </label>
          </div>
        </section>
      </section>
    </div>
    <div
      className="Skills__details"
    >
      <section
        className="Skills__wrapper"
        style={
          Object {
            "height": "100%",
            "transform": "translateX(-0%)",
            "transition": "transform 1s ease",
            "width": "100%",
          }
        }
      >
        <section
          className="Skills__detail Skills__detail__javascript Skills__detailSelected"
          style={
            Object {
              "left": "0%",
            }
          }
        >
          <button
            aria-label="menu"
            className="Hamburger__button"
            onClick={[Function]}
          >
            <span
              className="Hamburger"
            >
              <span
                className="Hamburger__inner"
              />
            </span>
          </button>
          <div
            className="Skills__stuff"
          >
            <p
              className="Skills__detail__subtitle"
            >
              About 
              JavaScript
            </p>
            <h2
              className="Skills__detail__title"
            >
              The Swiss Army knife of software development.
            </h2>
            <p
              className="Skills__detail__desc"
            >
              This incredibly important little buddy allows me to do everything from crafting experiences and defining behavior for the web to writing and running interactive servers.
            </p>
          </div>
        </section>
      </section>
    </div>
  </section>
  <section
    className="Stints"
    id="experience"
  >
    <div
      className="Stints__details"
    >
      <section
        className="Stints__wrapper"
        style={
          Object {
            "height": "100%",
            "transform": "translateX(-0%)",
            "transition": "transform 1s ease",
            "width": "100%",
          }
        }
      >
        <section
          className="Stints__detail Stints__detail__test"
          style={
            Object {
              "left": "0%",
            }
          }
        >
          <button
            aria-label="menu"
            className="Hamburger__button"
            onClick={[Function]}
          >
            <span
              className="Hamburger"
            >
              <span
                className="Hamburger__inner"
              />
            </span>
          </button>
          <div
            className="Stints__stuff"
          >
            <hgroup>
              <h2
                className="Stints__detail__title"
              >
                Test
              </h2>
              <h3
                className="Stints__detail__desc"
              >
                Senior Test Engineer
              </h3>
            </hgroup>
            <time
              className="Stints__detail__epoch"
            >
              July 2020 - present
            </time>
            <ul
              className="Stints__detail__highlights"
            >
              <li
                className="Stints__detail__highlight"
              >
                <svg
                  className="Stints__detail__highlight__icon"
                  fill="none"
                  height="16"
                  viewBox="0 0 11 13"
                  width="24.42"
                >
                  <title>
                    caret
                  </title>
                  <path
                    d="M8.95837 8.88236C10.3138 7.68793 10.3138 5.57523 8.95837 4.3808L6.48343 2.19984C4.54592 0.492473 1.5 1.86816 1.5 4.45062L1.5 8.81254C1.5 11.395 4.54592 12.7707 6.48343 11.0633L8.95837 8.88236Z"
                    fill="none"
                    stroke="#99F0FF"
                    strokeWidth={2}
                  />
                </svg>
                Write tested code mf.
              </li>
            </ul>
          </div>
        </section>
      </section>
    </div>
    <div
      className="Stints__list"
    >
      <section
        className="Stints__list__inner"
      >
        <h1
          className="Stints__title"
          data-label="03."
        >
          Experience.
        </h1>
        <section
          className="Entries"
        >
          <div>
            <input
              checked={false}
              className="EntryCard__radio"
              id="entry-test"
              onChange={[Function]}
              type="radio"
            />
            <label
              className="EntryCard"
              htmlFor="entry-test"
            >
              <section>
                <h4
                  className="EntryCard__title"
                >
                  Test
                </h4>
                <a
                  className="EntryCard__link with-underline"
                  href="#"
                >
                  Read more
                </a>
              </section>
              <img
                className="Stints__icon"
                src="/media/test.png"
              />
            </label>
          </div>
        </section>
      </section>
    </div>
  </section>
  <section
    className="Contact"
    id="contact"
  >
    <section
      className="container container--fullHeight"
    >
      <div
        className="landing__Contact"
      >
        <section>
          <h1
            className="Contact__title"
            data-label="04."
          >
            Let's chat.
          </h1>
          <p
            className="Contact__intro"
          >
            Got a question or proposal for me? Let’s get in touch! I have a personal rule of returning responses within a 3-hour window.
          </p>
          <p
            className="Contact__intro"
          >
            Start by
             
            <a
              className="with-underline"
              href="mailto:#"
            >
              saying hi
            </a>
             
          </p>
          <section
            className="Contact__footer"
          >
            <a
              className="Contact__footer__link"
              href="https://twitter.com/#"
            >
              <svg
                className="icon"
                viewBox="0 0 26 28"
              >
                <title>
                  twitter
                </title>
                <path
                  d="M25.312 6.375c-0.688 1-1.547 1.891-2.531 2.609 0.016 0.219 0.016 0.438 0.016 0.656 0 6.672-5.078 14.359-14.359 14.359-2.859 0-5.516-0.828-7.75-2.266 0.406 0.047 0.797 0.063 1.219 0.063 2.359 0 4.531-0.797 6.266-2.156-2.219-0.047-4.078-1.5-4.719-3.5 0.313 0.047 0.625 0.078 0.953 0.078 0.453 0 0.906-0.063 1.328-0.172-2.312-0.469-4.047-2.5-4.047-4.953v-0.063c0.672 0.375 1.453 0.609 2.281 0.641-1.359-0.906-2.25-2.453-2.25-4.203 0-0.938 0.25-1.797 0.688-2.547 2.484 3.062 6.219 5.063 10.406 5.281-0.078-0.375-0.125-0.766-0.125-1.156 0-2.781 2.25-5.047 5.047-5.047 1.453 0 2.766 0.609 3.687 1.594 1.141-0.219 2.234-0.641 3.203-1.219-0.375 1.172-1.172 2.156-2.219 2.781 1.016-0.109 2-0.391 2.906-0.781z"
                />
              </svg>
              <span
                className="with-underline ml-5"
              >
                twitter.com/
                #
              </span>
            </a>
            <p>
              Designed & developed by Kay Mathew
            </p>
          </section>
        </section>
      </div>
    </section>
  </section>
  <div
    className="landing__overlay"
    onClick={[Function]}
  />
</section>
`;
